### PR TITLE
Add BLS benchmarks

### PR DESCRIPTION
**Pull request by StephenButtolph**
## Why this should be merged

Better observability into the cost of BLS aggregation + verification

## How this works

Adds a benchmark

## How this was tested

running the bencharm

### CODE REVIEW

File: utils/crypto/bls/bls_benchmark_test.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+func BenchmarkAggregatePublicKeys(b *testing.B) {
+	keys := make([]*PublicKey, 4096)
+	for i := 0; i < 4096; i++ {
+		privateKey, err := NewSecretKey()
+		require.NoError(b, err)
+
+		keys[i] = PublicFromSecretKey(privateKey)
+	}
+
```
Comment (danlaine) [inline utils/crypto/bls/bls_benchmark_test.go:77]: Do we want to reset the timer after the setup?

File: utils/crypto/bls/bls_benchmark_test.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+			for n := 0; n < b.N; n++ {
+				require.True(Verify(publicKey, signature, message))
+			}
+		})
+	}
+}
+
+func BenchmarkAggregatePublicKeys(b *testing.B) {
+	keys := make([]*PublicKey, 4096)
+	for i := 0; i < 4096; i++ {
```
Comment (danlaine) [inline utils/crypto/bls/bls_benchmark_test.go:71]: nit
```suggestion
	for i := 0; i < len(keys); i++ {
```

**Review by aaronbuchwald — APPROVED** [review]
🤝

**Comment by StephenButtolph** [comment]
`Kurtosis Tests Basic / run_kurtosis_tests` is currently failing because the docker image is based on a version of avalanchego with updated compatibility requirements... So that's expected.

**Pull request merged**